proptest/tests/pass/with_params.rs

```diff
@@ -0,0 +1,21 @@
+fn main() {}
+
+#[proptest::property_test(
+    config = proptest::test_runner::Config {
+        cases: 10,
+        ..Default::default()
+    }
+)]
+fn no_trailing_comma(x: i32) {
+    assert_eq!(x, x);
+}
+
+#[proptest::property_test(
+    config = proptest::test_runner::Config {
+        cases: 10,
+        ..Default::default()
+    }
+)]
+fn trailing_comma(x: i32) {
```

Comment:
the test says `trailing_comma` but I don't see where the trailing comma is being tested? is that meant to be on `Default::default()`?